travis/script-gradle

```diff
@@ -19,4 +19,5 @@ esac
 $headless ./gradlew --continue --no-build-cache --stacktrace \
 	  "$SUBMODULE_PREFIX"build \
 	  ${documentation:+$SUBMODULE_PREFIX$documentation} \
-	  lintGradle
+	  lintGradle \
+	  verifyGoogleJavaFormat
```

Comment:
Can we list `verifyGoogleJavaFormat` first here?  It's a faster check than the `build` tasks, and this way you won't have to wait all the way through execution of regression tests to see that there is a formatting issue.  (I think on CI Gradle runs these tasks in the order they are listed after running dependent tasks.)

Reply:
You can see [here](https://travis-ci.org/wala/WALA/jobs/507474346) that with the current ordering `verifyGoogleJavaFormat` is run last

Reply:
Good suggestion. Thanks! Enacted in 8f18698ec9d0a55aef457dae471c251de17947bb.